src/layers/ethernet/mod.rs

```diff
@@ -14,34 +14,55 @@ use crate::{Layer, Packet, ENCAP_TYPE_ETH};
 
 pub const ETH_HEADER_LENGTH: usize = 14_usize;
 
+/// A Map maintaining EtherType -> Creator fns for Layer Creators of L3 Layers.
+///
+/// The creator function simply creates a `default` L3 struct that implements the dissector
+/// for the Layer.
 pub fn get_ethertypes_map() -> &'static RwLock<HashMap<EtherType, LayerCreatorFn>> {
-    /// A Map maintaining EtherType -> Creator fns for Layer Creators of L3 Layers.
-    ///
-    /// The creator function simply creates a `default` L3 struct that implements the dissector
-    /// for the Layer.
-    pub(crate) static ETHERTYPES_MAP: OnceLock<RwLock<HashMap<EtherType, LayerCreatorFn>>> =
-        OnceLock::new();
+    static ETHERTYPES_MAP: OnceLock<RwLock<HashMap<EtherType, LayerCreatorFn>>> = OnceLock::new();
     ETHERTYPES_MAP.get_or_init(|| RwLock::new(HashMap::new()))
 }
 
+/// A Map maintaining String -> EtherType of L3 Layers.
```

Comment:
Add more description for the map as a documentation. If we simply read this documentation it is not clear why this map exists and how it is to be used. For example something like this. 

```
/// This map will be used during sculpting a packet. For example for IPv4 you the entry
/// in the map will look like `ip -> 0x0800`
```